specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/virtualNetworkGateway.json

```diff
@@ -4480,6 +4492,39 @@
         "customBgpIpAddress"
       ],
       "description": "GatewayCustomBgpIpAddressIpConfiguration for a virtual network gateway connection."
+    },
+    "ConnectionAuthenticationType": {
+      "type": "string",
+      "description": "Gateway connection authentication type.",
+      "enum": [
+        "PSK",
+        "Certificate"
+      ],
+      "x-ms-enum": {
+        "name": "ConnectionAuthenticationType",
+        "modelAsString": true
+      }
+    },
+    "CertificateAuthentication": {
+      "type": "object",
+      "properties": {
+        "outboundAuthCertificate": {
```

Comment:
>outboundAuthCertificate [](http://example.com/codeflow?start=9&length=23)

If this is a secret, it needs to be tagged with x-ms-secret and is not allowed in GET payloads (must be nulled/blanked). If it's just ID info of the cert, consider changing the property name to distinguish what it's value contains.

Reply:
Added uri format and fixed the description.